### PR TITLE
Dcu regression

### PR DESCRIPTION
**Pull request by SchumBlubBlub**
Test for https://github.com/radare/radare2/pull/6736

### CODE REVIEW

File: t.archos/Linux-x86_64/dbg_step [hunk 1]
```diff
@@ -217,3 +217,14 @@ dr rip
 EXPECT='0x00400582
 '
 run_test
+
+NAME='dbg.dcu can reach main'
+FILE=../../bins/elf/analysis/calls_x64
+ARGS='-a x86 -d'
+BROKEN=
+CMDS='dcu main; ?v $$~[1];
```
Comment (radare) [inline t.archos/Linux-x86_64/dbg_step]: use multiple lines. its more readable than ;

File: t/cmd_write [hunk 2]
```diff
@@ -216,26 +216,3 @@ EXPECT='3412
 78563412
 '
 run_test
-
```
Comment (radare) [inline t/cmd_write]: why remove this?

Comment (SchumBlubBlub) [inline t/cmd_write]: mistake. I reverted locally. But is broken.

https://github.com/radare/radare2-regressions/pull/680#issuecomment-277743690

**Comment by SchumBlubBlub** [comment]
Fixed review.

**Comment by Maijin** [comment]
@SchumBlubBlub any plan to join us on irc/telegram ? That would ease a LOT the communication.

**Comment by Maijin** [comment]
https://telegram.me/joinchat/ACR-FkEK2owJSzMUYjt_NQ (better)
Or
#radare on IRC

**Pull request merged**